tests/models/test_horovod.py

```diff
@@ -371,7 +371,7 @@ def training_epoch_end(self, outputs) -> None:
 
 # todo: need to be fixed :]
 @pytest.mark.skip(reason="TODO: CI agent.jobstatus=Succeeded: Permission denied")
-@RunIf(skip_windows=True, horovod=True, num_gpus=2)
+@RunIf(horovod=True, skip_windows=True, num_gpus=2)
```

Comment:
just curious, is there any special reason for this ordering change?
without it the diff would be much smaller and easier to digest... :rabbit:

Reply:
found answer in https://github.com/PyTorchLightning/pytorch-lightning/pull/12432#discussion_r833847774